io_unix.go

```diff
@@ -47,7 +48,10 @@ func NewPipeIO(uid, gid int, opts ...IOOpt) (i IO, err error) {
 		}
 		pipes = append(pipes, stdin)
 		if err = unix.Fchown(int(stdin.r.Fd()), uid, gid); err != nil {
-			return nil, errors.Wrap(err, "failed to chown stdin")
+			// XXX: can't chown to pipe on darwin
```

Comment:
Suggest using this skip with a warning or debug, and wrap the err: https://github.com/containerd/containerd/pull/4526/commits/55d95502e07f7bd790c4ec9b4312996d995c07b3#diff-b3363345815845245ac7ce3ebbab476bR52-R57

for the comment suggest: `TODO: revert with proper darwin solution,  skipping for now as darwin chown is returning EINVAL on anonymous pipe`

Reply:
updated with your suggestion. and also updated go.mod as it now uses logrus package.